static/js/zcommand.js

```diff
@@ -87,6 +88,31 @@ exports.process = function (message_content) {
         return true;
     }
 
+    if (tokens[0] === '/subscribe') {
+        tokens[0] = tokens[0].replace('/', '');
+
+        exports.send({
+            command: tokens.join(' '),
+            on_success: function (data) {
+                if (data.stream && data.user_id) {
+                    console.log(data.stream, data.user_id);
```

Comment:
not needed

Reply:
Left this in, in a hurry! I have removed it.